tap_fullstory/schemas/events.json

```diff
@@ -26,6 +26,18 @@
     "EventTargetSelectorTok": {
         "type": ["null", "string"]
     },
+    "EventModFrustrated": {
+      "type": ["null", "integer"]
+    },
+    "EventModDead": {
+      "type": ["null", "integer"]
+    },
+    "EventModError": {
+      "type": ["null", "integer"]
+    },
+    "EventModSuspicious": {
+      "type": ["null", "integer"]
+    },
```

Comment:
I was confused about this at first, but to be clear, Tim is referring to a gist containing the logs of the pre-and-post fix runs of the tap.

Reply:
Thanks, I was confused at first as well. Are you guys wanting a gist that shows all the fields stripped and not stripped, or just these four "EventMod" fields. Their absence from the original version of the tap was kind of conspicuous, so I was wondering if there's some known reason they were left out?

Reply:
Just the fields that are being added in this PR will be fine. This is the best way we currently have to make sure that schema changes mesh well with the transformation code.

Looking closer that this tap, it doesn't look like there's any integer type conversions or schema validation in the tap itself, whereas more full-featured taps use the `transformer` in the `singer-python` library to check schema before emitting records. So, we'd be looking to make sure that these fields are being sent out as JSON integers to support targets and other processes down the pipeline who will do more schema checking themselves.

Unfortunately, I don't have context on the original reason these were left out. It could just be an oversight by the original implementer.